package.json

```diff
@@ -84,6 +84,7 @@
     "chokidar": "^3.3.1",
     "coveralls": "^3.0.9",
     "electron": "8.0.2",
+    "electron-rebuild": "^1.11.0",
```

Comment:
Where is this directly used? All I see is an addition to Forge's `electronRebuildConfig`, and Forge already depends on `electron-rebuild`.

Reply:
the version there doesnt include the latest release, which has the ABI override functionality - i can bump it in forge though and then bump for here if you'd prefer

Reply:
I think you can just run `npm upgrade --depth 100 electron-rebuild` or something. _(This is also known as the "why isn't node-abi updated" problem :sob:)_

Reply:
> This is also known as the "why isn't node-abi updated" problem 😭

Maybe time for additional documentation about that in `electron-rebuild` readme?

Reply:
I think the error just needs to be clarified.

Reply:
@malept can you expand? the issue here is that there is no _assigned_ ABI number, and so one needs to force one.

Reply:
Sorry, my point was that `electron-rebuild` is a transitive dependency of Fiddle, so it's not necessary to add it as an explicit dependency just for the purposes of forcing a version bump.

I only mention `node-abi` because Electron Forge has the same transitive dependency problem with that module.